src/backend/parser/cypher_clause.c

```diff
@@ -4837,7 +4871,9 @@ static Expr *transform_cypher_node(cypher_parsestate *cpstate,
     return expr;
 }
 
-static Node *make_edge_expr(cypher_parsestate *cpstate, RangeTblEntry *rte)
+static Node *make_edge_expr(cypher_parsestate *cpstate,
+                            ParseNamespaceItem *pnsi,
+                            char *label)
```

Comment:
Why is label parameter added here. I can't see its use in the function. Is there any specific reason I am overseeing ?

Reply:
Was an oversight on my part, fixed.